tasks/main.yml

```diff
@@ -1,280 +1,13 @@
----
-# TODO:
-#  - use include_tasks to group them
-- name: Install dependencies
-  apt: pkg={{item}} state=latest
-  with_items:
-    - curl
-    - sudo
-    - unzip
-    - vim
-    - nginx
-    - ffmpeg
-    - postgresql
-    - openssl
-    - g++
-    - make
-    - redis-server
-    - git
-    - python-psycopg2
-    - build-essential # for bcrypt?
-    - apt-transport-https # for node apt repository
-    - python-ruamel.yaml
 
-# Configure nodejs
-- name: Install NodeJS apt key
-  apt_key:
-    url: https://deb.nodesource.com/gpgkey/nodesource.gpg.key
-    state: present
+- include_tasks: install_dependencies.yml
 
-- name: Add NodeJS apt repository
-  apt_repository:
-    repo: deb https://deb.nodesource.com/node_9.x "{{ ansible_distribution_release }}" main
-    state: present
-    filename: nodesource
+- include_tasks: postgres.yml
 
-- name: Install NodeJS
-  apt:
-    update_cache: yes
-    pkg: nodejs
-    state: latest
+- include_tasks: nginx.yml
```

Comment:
First certbot.yml, then nginx.yml, otherwise Nginx cannot read ssl certificates generated

Reply:
It can't because the certbot task adds the location of the certificate to the nginx configuration. If we install cerbot first, it won't find the nginx config file. [Here](https://github.com/UdelaRInterior/ansible-peertube/blob/30cc6d71a02345641cded175e478fed7221f472a/tasks/certbot.yml#L24)